scripts/scriptutils/part_retrieval.py

```diff
@@ -1,434 +1,454 @@
-import io
```

Comment:
Something appear to have reformatted this file (change in newlines maybe?) such that it appears to entirely change. Please undo whatever formatting change happened, so that only the actual diff appears. Otherwise the change cannot be reviewed.

Reply:
I believe I had reformatted to CRLF (without knowing), so I reverted back to LF. Let's see if this fixes things.